src/message.ts

```diff
@@ -403,6 +403,37 @@ export class Message implements Sayable {
     return fromId === userId
   }
 
+  /**
+   * Get the mentioned contact which the message is mentioned for.
+   * @returns {Contact[]} return the contactList which the message is mentioned for
+   */
+  public mention(): Contact[] {
+    let contactList: Contact[] = []
+    const room = this.room()
+    if (!room) return contactList
+    if (this.type() !== MsgType.TEXT) return contactList
```

Comment:
Please always use `{}` for `if`, even if there's only one line of code, because that will make code more readable.

```ts
if (this.type() !== MsgType.TEXT
    || !this.room) {
    return contactList
}
```